app/scripts/modules/amazon/src/securityGroup/configure/EditSecurityGroupCtrl.js

```diff
@@ -90,6 +90,7 @@ module.exports = angular
       let command = {
         credentials: group.accountName,
         name: group.name,
+        securityGroupName: group.name,
```

Comment:
@christopherthielen : not sure if there are other places I need to change

Reply:
unfortunately, yes.   look for usages of  `SecurityGroupWriter.upsertSecurityGroup`

Reply:
interestingly, some call sites already use `securityGroupName`, i.e.: 

https://github.com/spinnaker/deck/blob/master/app/scripts/modules/openstack/src/securityGroup/configure/wizard/upsert.controller.js#L161-L164

https://github.com/spinnaker/deck/blob/master/app/scripts/modules/kubernetes/src/securityGroup/configure/wizard/upsert.controller.js#L166-L169

Reply:
🤕 the deleteTask expects this format but these are both for upsert..